GOT-StudentVersion/GOT-StudentVersion/GOTEpisdoe.swift

```diff
@@ -28,8 +28,20 @@ class GOTEpisode {
         self.summary = summary
         self.mediumImageID = mediumImageID
         self.originalImageID = originalImageID
+        
     }
-    static let allEpisodes = [
+    
+    enum Seasons: String {
```

Comment:
I'm not sure how necessary this enum was. It is an interesting idea, but maybe not really needed